### ISSUE TITLE
MMSegmentation Tutorial.ipynb ImportError: /usr/local/lib/python3.7/dist-packages/mmcv/_ext.cpython-37m-x86_64-linux-gnu.so: undefined symbol: _ZNK2at6Tensor7is_cudaEv

### ISSUE THREAD

**Issue by sabernn** [open-mmlab/mmsegmentation#1300]
Hi,

I'm trying to run MMSegmentation Tutorial.ipynb on Google COLAB. When I run this line:

```none
from mmseg.apis import inference_segmentor, init_segmentor, show_result_pyplot
```

I get this error:

ImportError: /usr/local/lib/python3.7/dist-packages/mmcv/_ext.cpython-37m-x86_64-linux-gnu.so: undefined symbol: _ZNK2at6Tensor7is_cudaEv

Thanks,
Saber

**Comment by MengzhangLI** [comment]
Hi, I think it is probably caused by incorrect version of torch and cuda. 

You can check out version of torch in your python envronment and cuda version using `nvidia-smi`. Then first `pip uninstall mmcv` and `pip uninstall mmcv-full`, then pip install correct mmcv again:

https://github.com/open-mmlab/mmcv#installation

**Comment by mfernezir** [comment]
Current Google Colab uses a different CUDA version. You can see it after running `!nvcc -V` in the first cell:
```
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2020 NVIDIA Corporation
Built on Mon_Oct_12_20:09:46_PDT_2020
Cuda compilation tools, release 11.1, V11.1.105
Build cuda_11.1.TC455_06.29190527_0
```
The second cell in the notebooks downloads the version for CUDA 10.1. You can fix it by downloading the 11.1 version. I also switched to newer PyTorch:

```
# FIX PYTORCH VERSIONS to Colab Cuda
# Install PyTorch
!conda install pytorch=1.10.0 torchvision cudatoolkit=11.1 -c pytorch
# Install MMCV
!pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu111/torch1.10/index.html
```
